### PR TITLE
[eclipse/xtext#1160] Update copyright date range

### PR DESCRIPTION
**Pull request by kthoms**
Additional file cleanup
- remove empty line at end of file

Signed-off-by: Karsten Thoms <karsten.thoms@itemis.de>

### CODE REVIEW

File: org.eclipse.xtend.caliper.tests/src/org/eclipse/xtend/caliper/tests/AbstractXtendTestCase.java [hunk 1]
```diff
@@ -171,4 +171,4 @@ protected XtendField field(String string) throws Exception {
 		return (XtendField) clazz.getMembers().get(0);
 	}
 
-}
\ No newline at end of file
+}
```
Comment (cdietrich) [inline org.eclipse.xtend.caliper.tests/src/org/eclipse/xtend/caliper/tests/AbstractXtendTestCase.java]: intended?

Comment (kthoms) [reply org.eclipse.xtend.caliper.tests/src/org/eclipse/xtend/caliper/tests/AbstractXtendTestCase.java]: This happens when using of Files.readAllLines()...Files.write()
I'm afraid I can't change that except for manually reverting these lines. But would happen the next time again.

**Comment by kthoms** [comment]
4 tests are failing:
- org.eclipse.xtend.ide.tests.RenameJavaClassSwtBotTest.org.eclipse.xtend.ide.tests.RenameJavaClassSwtBotTest
- org.eclipse.xtend.ide.tests.RenameXtendClassBug516096SwtBotTest.org.eclipse.xtend.ide.tests.RenameXtendClassBug516096SwtBotTest
- org.eclipse.xtend.ide.tests.RenameXtendClassSwtBotTest.org.eclipse.xtend.ide.tests.RenameXtendClassSwtBotTest
- org.eclipse.xtend.ide.tests.contentassist.Issue242Test.test_01

**Comment by cdietrich** [comment]
these are all flaky tests. please create a separate issue

**Comment by cdietrich** [comment]
+1 add XtextGrammarRefactoringIntegrationTest

Unfortunately i never could reproduce any flakyness on any of my machines.
mac / windows / manjaro linux

**Pull request merged**